src/is-authenticated-with-context.js

```diff
@@ -0,0 +1,10 @@
+import Cookies from 'js-cookie'
+
+export default function () {
```

Comment:
I thought you would want to pass in the context you care about to determine if the user is authenticated to that specific context.